loader/src/main/java/net/neoforged/fml/loading/FMLConfig.java

```diff
@@ -64,11 +69,20 @@ void buildConfigEntry(ConfigSpec spec, CommentedConfig commentedConfig) {
         }
 
         @SuppressWarnings("unchecked")
-        private <T> T getConfigValue(CommentedFileConfig config) {
+        private <T> T getConfigValue(CommentedConfig config) {
             return (T) this.entryFunction.apply(config != null ? config.get(this.entry) : this.defaultValue);
         }
 
+        /**
+         * @deprecated Use {@link FMLConfig#updateConfig(ConfigValue, Object)} instead.
+         */
+        @Deprecated(forRemoval = true)
```

Comment:
We can remove it now

Reply:
We can but you'll have to reintroduce it when backporting to 1.21.1. I thought we might as well wait. 😅

Reply:
Ah right, fair enough.